kingpin/actors/aws/iam.py

```diff
@@ -27,6 +27,7 @@
 from kingpin.actors.aws import base
 from kingpin.actors import exceptions
 from kingpin.constants import REQUIRED
+from kingpin.constants import STATE
```

Comment:
Not sure how I like the kingpin-global STATE with 2 valid values. Wouldn't it be easier to just list the allowed values as the first entry in the all_options? Example:

```
'state': (('present','absent'), 'present', 'Desired state of the User: present/absent'),
```

Other actors may have other states, like 'public','private' etc